optimum/exporters/openvino/convert.py

```diff
@@ -380,6 +409,13 @@ def ts_patched_forward(*args, **kwargs):
             logger.warning(f"Export model to OpenVINO directly failed with: \n{ex}.\nModel will be exported to ONNX")
             if patch_model_forward:
                 model.forward = orig_forward
+            if stateful:
+                # cannot raise because stateful is enabled by default and it would break backward compatibility for models that couldn't convert to OV directly
+                # TODO: Implement stateful for ONNX path as well, not doing it right now because of lack of validation
+                logger.warn(
+                    "[ WARNING ] Making stateful models is not supported when exporting to ONNX as an intermediate step. A stateless model will be exported instead. It may result in sub-optimal inference performance."
+                    "Provide a model that can be converted to OpenVINO without fallback to ONNX conversion path."
```

Comment:
```suggestion
                    "Provide a model that can be converted to OpenVINO without fallback to ONNX conversion path."
```

I think most people will need to export the model they are exporting and "provide another model" is then not really helpful. In my experience, when export with PyTorch failed and ONNX was used instead, it was usually because of a technical issue, with optimum-intel or dependencies. So we could suggest to upgrade dependencies with `pip install --upgrade --upgrade-strategy eager optimum[openvino]` and maybe suggest to create an issue if exporting with PyTorch still fails?